src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.targets

```diff
@@ -47,6 +47,9 @@ Copyright (c) .NET Foundation. All rights reserved.
   
   <!-- Import targets from NuGet.Build.Tasks.Pack package/Sdk -->
   <PropertyGroup Condition="'$(NuGetBuildTasksPackTargets)' == '' AND '$(ImportNuGetBuildTasksPackTargetsFromSdk)' != 'false'">
+    <IncludeBuildOutput Condition=" '$(PackAsTool)' == 'true' ">false</IncludeBuildOutput>
```

Comment:
HELP NEEDED: I don't want to put logic here. But these 2 need to be evaluated after user's csproj and before NuGet pack.